supporting_components/graph.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, graph: "Graph", label=None, graph_label=None):
         self.graph_label = graph_label
         self._incidence = {}
         self.twin_degree = None
+        self.colornum = None
```

Comment:
Waarvoor is deze?

Reply:
In de test van isomorphism_problem kopieer ik de grafen voordat ze een kleur hebben. En omdat colornum wel in de copy() methode staat, gaat dit niet goed. Dus heb voor nu alle properties die in copy() staan ook hier een initiele waarde gegeven.